crates/sui-authority-aggregation/src/lib.rs

```diff
@@ -39,7 +39,8 @@ pub async fn quorum_map_then_reduce_with_timeout_and_prefs<
     initial_state: S,
     map_each_authority: FMap,
     reduce_result: FReduce,
-    initial_timeout: Duration,
+    max_timeout: Duration,
+    min_timeout: Option<Duration>,
```

Comment:
rename to `prefetch_timeout` and add comments what it does, .e.g it's included in `max_timeout`?
In fact we should probably rename `max_timeout` to `total_timeout`, `max` can be confusing